ledger/participant-integration-api/src/main/scala/platform/usermanagement/CachedUserManagementStore.scala

```diff
@@ -57,7 +57,9 @@ class CachedUserManagementStore(
   }
 
   override def createUser(user: User, rights: Set[domain.UserRight]): Future[Result[Unit]] =
-    delegate.createUser(user, rights)
+    delegate
```

Comment:
@simonmaxen-da I suspect there is a better place to test this than daml script, any pointers?

Reply:
+1

Reply:
There is a `CachedUserManagementStoreSpec`

Reply:
Here's a PR with a test case: https://github.com/digital-asset/daml/pull/12606

Reply:
Do you want to merge this and then the testcase, merge the test case first as expected failure or cherrypick either the testcase into this or the other way around?

Reply:
Checked locally that the testcase fails without the fix. 
The simplest is probably to merge/cherry-pick/copy the test case into this PR and then merge this PR to main

Reply:
sgtm so let’s do that please @nickchapman-da

Reply:
I think the question above is better answered by @meiersi-da

Reply:
unit test merged in. thanks @pbatko-da